pom.xml

```diff
@@ -527,7 +527,7 @@ Import-Package: \\
         <plugin>
           <groupId>com.github.ekryd.sortpom</groupId>
           <artifactId>sortpom-maven-plugin</artifactId>
-          <version>3.4.1</version>
+          <version>4.0.0</version>
```

Comment:
Why do we use the `sortpom-maven-plugin`? Wouldn't it be easier to use something like

```
          <pom>
            <sortPom>
              <expandEmptyElements>false</expandEmptyElements>
              <indentSchemaLocation>true</indentSchemaLocation>
              <predefinedSortOrder>recommended_2008_06</predefinedSortOrder>
              <sortDependencyManagement/>
              <sortDependencyExclusions/>
              <sortProperties>true</sortProperties>
            </sortPom>
          </pom>
```

in the spotless configuration?

Reply:
I'd be happy to remove as many plugins as possible.

If I replace sortpom-maven-plugin as proposed, spotless step will automatically download
https://repo1.maven.org/maven2/com/github/ekryd/sortpom/sortpom-sorter/3.2.1/sortpom-sorter-3.2.1.jar and related files. If I remove everything and purge them from my m2 cache, no additional files will be downloaded, so it seems to be caused by the softpom section in spotless config.

Do you think we should proceed this way, effectively getting rid of one dependency we need to manage?

Reply:
It is on my todo list to use the spotless sortpom config. I think it previously didn't exist.

Reply:
You can use `<version>...</version>` in the `<sortPom>` config to specify a specific version. I think we can also remove the `eclipseWtp` section for the `pom.xml`.

Reply:
There's now https://github.com/openhab/openhab-core/pull/4618